src/mkdocs_bibtex/utils.py

```diff
@@ -309,3 +312,55 @@ def tempfile_from_url(name, url, suffix):
     raise RuntimeError(
         f"Couldn't successfully download the url: {url}"
     )  # pragma: no cover
+
+def tempfile_from_zotero(name: str, url: str, suffix: str) -> str:
```

Comment:
Rest of the file has two lines between functions. Keep formatting as consistent as possible for lowest friction. `requirements-testing.txt` does have Ruff@0.9.1 in it, can you lint/format using Ruff?

```suggestion


def tempfile_from_zotero(name: str, url: str, suffix: str) -> str:
```

Reply:
Formatting has been a true pain. The rest of the file is not formatted according to the ruff config so I had to be careful what I formatted. Looks like these new lines were missed out. If I tried to format more than the functions I wrote, the boundaries can make it hard to patch updates to prevent touching code that isn't mine

Reply:
Why tease us that they're using Ruff when they're not 🙃